pkg/kubectl/cmd/annotate_test.go

```diff
@@ -451,7 +451,7 @@ func TestAnnotateObject(t *testing.T) {
 	f, tf, codec, _ := cmdtesting.NewAPIFactory()
 	tf.Printer = &testPrinter{}
 	tf.UnstructuredClient = &fake.RESTClient{
-		APIRegistry:          api.Registry,
+		GroupVersion:         api.Registry.GroupOrDie(api.GroupName).GroupVersion,
```

Comment:
Is this fix in a separate commit. You've got lots of these lines and this update looks good.  We could split it out and merge it.

Reply:
yeah, it's in its own commit (https://github.com/kubernetes/kubernetes/pull/53743/commits/8f9abfd3b08c14db48c251c65edc3973658446b6)